core/shared/src/main/scala/spire/math/Rational.scala

```diff
@@ -839,7 +856,14 @@ private[math] trait RationalApproximateNRoot extends NRoot[Rational] {
     Rational(n.toDouble ** k.toDouble)
 }
 
-private[math] trait RationalIsField extends Field[Rational] {
+private[math] trait RationalIsField extends Field.FieldOfFractionsGCD[Rational, SafeLong] {
+  override def gcd(x: Rational, y: Rational)(implicit ev: Eq[Rational]) = x.gcd(y)
```

Comment:
Since `gcd` and `lcm` are being overridden, is there any benefit in extending `FieldOfFractionsGCD` here?